### DISCUSSION TITLE
Issues with using ref()

### DISCUSSION THREAD

**Discussion by Sibgat-Ul** [quasarframework/quasar#9023] [CLI - Capacitor / Cordova modes] [answered]
I was trying to use  **ref() as v-model on q-toggle**, to **switch between light and dark mode**, works fine on desktop and mobile if i use browser but, shows *object not extensible error on capacitor* mode/app.  Thanks in advance <3

**Comment by JokerMartini** [comment]
How are you switching the theme, im trying todo the same thing right now.

**Reply by rstoenescu** [reply]
@JokerMartini
* (v2) https://next.quasar.dev/quasar-plugins/dark
* (v1) https://quasar.dev/quasar-plugins/dark

**Comment by rstoenescu** [comment]
@Sibgat-Ul A codepen of what you are trying out would be nice.

**Comment by ontwikkelfabriek** [comment, marked as answer]
@Sibgat-Ul 
See this thread on the Capacitor github about dark mode not appearing. The last post is about a Quasar solution.
https://github.com/ionic-team/capacitor/discussions/1978#discussioncomment-728873
